README.md

```diff
@@ -18,27 +20,39 @@ Examples of unacceptable behavior by participants include:
 
 - The use of sexualized language or imagery and unwelcome sexual attention or advances
 - Trolling, insulting/derogatory comments, and personal or political attacks
-- Public or private harassment or insensitivity related to gender identity or expression, sexual orientation, disability, physical appearance, body size, race or religion
+- Public or private harassment or insensitivity related to age, gender identity or expression, sexual orientation, disability, physical appearance or body size, level of experience, education, socio-economic status, nationality, ethnicity or race, or religion
 - Publishing others’ private information, such as a physical or electronic address, without explicit permission
 - Other conduct which could reasonably be considered inappropriate in a professional setting
 
 ## Etiquette
+
 We encourage everyone to review and follow the [Slack Etiquette Guide](https://github.com/UtahJS/slack-coc/blob/master/ETIQUETTE.md).
 
+## Advertisements
```

Comment:
I like these additions - they keep the option for community members to look for work or provide opportunities in good faith, but also provide a clear course of action for moderators to prevent spammy behavior.